### ISSUE TITLE
"semicolon" rule does nothing in `declare class`

### ISSUE THREAD

**Issue by andy-ms** [palantir/tslint#2384]
### Bug Report

- __TSLint version__: 5.0.0-dev.0
- __TypeScript version__: 2.3.0-dev.20170321
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
declare class C {
    a(): void
}

declare interface I {
    a(): void
}
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "semicolon": [true]
    }
}
```

#### Actual behavior

Lint error in the interface.

#### Expected behavior

Lint error in the class, too.

**Comment by jmlopez-rod** [comment]
Same thing on `export`.

```ts
export {
  A,
  B
}
```

Do we need a semicolon there? Based on this it seems we do: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-exports

**Comment by ajafff** [comment]
Both should be fixed in master. 
Can you please try that and report if something is still missing.

**Comment by andy-ms** [comment]
Works great! Thanks!
